src/components/Table.jsx

```diff
@@ -522,28 +570,46 @@ export default function Table() {
       ) {
         setAssigneeType("editors");
       }
-      console.log(rowData);
       setIsModalVisible(true);
       setStyleId([rowData?.styleId]);
       setWorkflowId([rowData?.id]);
     }
     e.stopPropagation();
   };
 
-  const handleRowSelectIcons = (rowData, imgPath, type) => {
+  const handleRowSelectIcons = (rowData, type) => {
     return (
       <div
         onClick={(e) => iconClickHandler(e, type, rowData)}
         className="flex justify-content-end"
       >
         <span>
           {rowData.id == showEdit && isRowSelected && isAdmin && (
-            <button
+            <>
+             {type == "edit" ? 
+             <button
               className="bg-white flex rounded-full justify-center items-center border border-grey-30  h-[30px] w-[30px]"
               onClick={handleEditIcon}
             >
-              <img alt={`${imgPath} svg`} src={imgPath} />
+            <Tooltip target=".quick-fix"/>
+            <img alt={`${Edit} svg`} src={Edit} data-pr-tooltip="Quick Fix" data-pr-position="top" className="quick-fix"  style={{ fontSize: '200px', cursor: 'pointer' }}></img>
+            </button> 
+            :
+            <button
+            className="bg-white flex rounded-full justify-center items-center border border-grey-30  h-[30px] w-[30px]"
+            onClick={handleEditIcon}
+          >
+            <Tooltip target=".assign"/>
+            <img alt={`${ currentTab == "Assigned" || currentTab == "InProgress"
```

Comment:
Your prettier alignment is wrong.
Fix the vscode issue and make sure prettier works on save.